content/events/2020-paris/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Paris
+date: '2020-10-29'
+end: '2020-10-30'
+location:
+  country: France
+  city: Paris
+  url: https://en.wikipedia.org/wiki/Paris
+  venue: "????"
+map: paris-map.png
+sched_slug: kubernetescommunitydayparis
+social:
+  twitter: 'https://twitter.com/kubedaysparis'
+sponsors:
+- level: diamond
+  orgs:
+#  - name: OVH
+#    logo: ovh.svg
+#    url: https://en.wikipedia.org/wiki/OVH
+- level: platinum
+  orgs:
+- level: gold
+  orgs:
+speakers:
+# - name: Phippy
+#   image: phippy-color.png
+#   twitter: https://twitter.com/cloudnativefdn
+#   bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#   homepage: https://phippy.io
+organizers:
```

Comment:
It looks like some of the images aren't showing up and I know you are still adding some as this is WIP. Please let me know when complete and we can address if there is a bug.

Reply:
Please also update the email address on this page to your alias. Thanks!

Reply:
Sure, I already asked all the co-organizers to send me a profile picture to update the PR.

Reply:
Hi @jcsirot- Just checking in on this PR. Is it ready for review?

Reply:
Hi @KTan1226 The website is ok but we are just waiting for the a final confirmation of one the initial sponsors, which takes a little more time than expected due to the current lockdown in France. As soon as we receive the confirmation I will send you a message for the review.

Reply:
Thanks for the update @jcsirot!

Reply:
Hello @KTan1226 The PR is ready for review!

Reply:
Hi @jcsriot- looks good! Can you have another organizer give a LGTM? I can merge after that.

Reply:
@rbenzair please could you give a LGTM before merging